package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "styled-components",
-  "version": "3.1.6",
+  "name": "@marionebl/styled-components",
```

Comment:
Changes in name and version have been introduced accidentally into this branch, I assume?

Can you make sure to revert them please, so we can proceed to merge this PR?

Thanks for your effort btw! Such a legend! 💯